CHANGELOG.md

```diff
@@ -1,86 +0,0 @@
-# CNN Logger ChangeLog
```

Comment:
Why is this file being deleted?

Reply:
I deleted all of the cnn-logger files and copied over from my other repo. we can ad that guy back for the history. This was a total from scratch rewrite.